README.rst

```diff
@@ -348,120 +356,114 @@ For example:
-During step 1 of the ``cartridge build`` command, ``cartridge`` builds the application
-inside the application directory -- unlike when building the application as part
-of the ``cartridge pack`` command, when the application is built in a temporary
-`build directory <Build directory_>`_ and no build artifacts
+First, ``cartridge`` builds the application inside the application directory.
+This is different from ``cartridge pack``, which builds the application inside the
+`build directory <Build directory_>`_. No build artifacts
 remain in the application directory.
 
-During step 2 -- the key step here -- ``cartridge`` installs all dependencies
-specified in the rockspec file (you can find this file within the application
-directory created from template).
-
-(An advanced alternative would be to specify build logic in the
-rockspec as ``cmake`` commands, like we
-`do it <https://github.com/tarantool/cartridge/blob/master/cartridge-scm-1.rockspec#L26>`_
-for ``cartridge``.)
-
-If your application depends on closed-source rocks, or if the build should contain
-rocks from a project added as a submodule, then you need to **install** all these
-dependencies before calling ``tarantoolctl rocks make``.
-You can do it using the file ``cartridge.pre-build`` in your application root
-(again, you can find this file within the application directory created from template).
-In this file, you can specify all rocks to build
+Second, ``cartridge`` installs all dependencies specified in the rockspec file.
+(That file is located in the application directory created from template.)
+
+Alternatively, you can define the build logic in the rockspec in the form of ``cmake`` commands,
+`like we do <https://github.com/tarantool/cartridge/blob/master/cartridge-scm-1.rockspec#L26>`_
+in ``cartridge``.)
+
+If your application depends on closed-source rocks, or if your build contains
+rocks from a project added as a submodule, install all those
+dependencies **before** calling ``tarantoolctl rocks make``.
+You can do so using the file ``cartridge.pre-build`` in your application root
+(again, that file is located in the application directory created from template).
+In this file, you can specify all the rocks to build
 (e.g. ``tarantoolctl rocks make --chdir ./third_party/proj``).
-For details, see `special files <Special files_>`_.
+For details, see `build and packing files <Special files_>`_.
 
-As a result, in the application's ``.rocks`` directory you will get a fully built
-application that you can start locally from the application's directory.
+As a result, a fully built application will appear in the ``.rocks`` directory.
+You can start it locally from the application directory.
 
 .. _cartridge-cli-starting-stopping-an-application-locally:
 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Starting/stopping an application locally
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-**********
+*********
```

Comment:
We don't use `*` for headings. https://www.tarantool.io/en/doc/latest/contributing/docs/markup/headings/